### DISCUSSION TITLE
Color or Colour? Organize or Organise

### DISCUSSION THREAD

**Discussion by alevy** [tock/tock#3724] [Polls]
Tock is for everyone. _Most_ importantly, regardless of spelling preferences. However, in practice, some of us spell things differently. That's OK! We don't have to agree on whether a Z or an S makes a "zzz" sound, or whether it's Zee or Zed. We can even agree that both are correct. We can even agree that English, being derived from many disparate languages has no uniform set of rules and it's basically a free-for-all.

However, we do write documentation, name functions, modules, variables, etc together, and it _can_ be confusing to have a "Color" struct inside a "colours" module. It might make things more difficult to find if they are spelled differently in different places.

**Comment by wright-joe** [comment]
Is Chaous an option?

**Comment by Afrowave** [comment]
This is an interesting question. I have grown up writing "colour" but in my computer life, I write "color". Since English(US) leads the way in technology, "color" is fine. But for everyday writing, "organize" is out.😁
